docs/build/Build-for-ARTIK.md

```diff
@@ -9,6 +9,10 @@
   
   This is required to get rootstrap for Tizen (set of native libraries).
 * arm-linux-gnueabi-gcc cross compiler (can be found in Tizen Studio / Native toolchain)
+* Otherwise, you can install on your PC.
```

Comment:
I think the sentence "Otherwise, ..." is not the same level with artik050 and tizen studio, but is continued information for cross compiler `arm-linux-gnueabi-gcc`. What about removing '*' and make the sentence indented.

LGTM. Except for comment above.